src/mbgl/util/chrono.cpp

```diff
@@ -22,8 +22,13 @@ std::string rfc1123(Timestamp timestamp) {
     std::time_t time = std::chrono::system_clock::to_time_t(timestamp);
     std::tm info;
     _gmtime(&time, &info);
-    char buffer[30];
-    snprintf(buffer, 30, "%s, %02d %s %4d %02d:%02d:%02d GMT", week[info.tm_wday], info.tm_mday,
+
+    // Buffer size 30 is OK assuming the year has 4 digits. However, In theory, it might have
+    // more digits. Under gcc 8.3.0 with -Os optimization flag, there is compiler warning
+    // complaining about the buffer size might be too small. Inceasing the buffer to 32 fixes
+    // the warning.
+    char buffer[32];
+    snprintf(buffer, 32, "%s, %02d %s %4d %02d:%02d:%02d GMT", week[info.tm_wday], info.tm_mday,
```

Comment:
why copy 32 symbols, given the actual data takes 30?

Reply:
The second parameter is the maximum number of bytes to be used in the buffer. It doesn't mean 32 bytes will be copied. Normally, it should be consistent with the buffer size.

Reply:
@pozdnyakov , any update? Can we merge this change?

Reply:
Sorry for the delay, LGTM, thanks!